CMakeLists.txt

```diff
@@ -40,9 +40,9 @@ option(LOGGING
 option(SIMULATE
     "Build enclave in simulation mode" OFF)
 option(OBLIVIOUS
-    "Enable oblivious training and inference" ON)
+    "Enable oblivious training and inference" OFF)
 option(USE_AVX2 
-    "Use AVX2 instructions to speed up oblivious primitives" ON)
+    "Use AVX2 instructions to speed up oblivious primitives" OFF)
```

Comment:
Can we turn this on by default? Then someone who wants obliviousness will only need to change one option instead of two. If obliviousness is disabled this shouldn't affect anything right?